src/services/projects.js

```diff
@@ -383,8 +390,21 @@ module.exports = class ProjectsHelper {
 					}
 				}
 
+				if (fieldData && Object.keys(entityType.validations).includes('max_length')) {
+					const max_text_length = entityType.validations.max_length
+					const checkLength = utils.lengthChecker(max_text_length, fieldData.length)
```

Comment:
@adithyadinesh0412 change checkLength to isLengthValid and lengthChecker to compareLength